logistics_project/apps/malawi/tests/product.py

```diff
@@ -4,15 +4,17 @@
 from logistics_project.apps.malawi.tests.util import create_hsa
 from logistics_project.apps.malawi.tests.base import MalawiTestBase
 from django.conf import settings
+from static.malawi.config import BaseLevel, SupplyPointCodes
+
 
 class TestProductLevels(MalawiTestBase):
     
-    def testEmergencyLevels(self):
+    def testStaticLoader(self):
         csv_file = open(settings.STATIC_PRODUCTS, 'r')
         try:
             count = 0
-            expected_count = Product.objects.exclude(emergency_order_level=None).count()
-            static_sp = SupplyPoint.objects.all()[0]
+            # EPI Products aren't loaded through the static loader so exclude them here
```

Comment:
the current static loader doesn't support the product split and also seems to be loading old stale data.  i didn't have time to fix all of that so opted to just make a management command which we can use to keep the epi products up to date and which is shared with the tests. but i have it on my list to make that more consistent.